test-env/server/boot/explorer.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function mountLoopBackExplorer(server) {
+  var explorer;
+  try {
+    explorer = require('loopback-explorer');
+  } catch(err) {
+    // Print the message only when the app was started via `server.listen()`.
+    // Do not print any message when the project is used as a component.
+    server.once('started', function(baseUrl) {
+      console.log(
+        'Run `npm install loopback-explorer` to enable the LoopBack explorer'
+      );
+    });
+    return;
+  }
+
+  var restApiRoot = server.get('restApiRoot');
+
+  var explorerApp = explorer(server, { basePath: restApiRoot });
+  server.use('/explorer', explorerApp);
+  server.once('started', function() {
+    var baseUrl = server.get('url').replace(/\/$/, '');
+    // express 4.x (loopback 2.x) uses `mountpath`
+    // express 3.x (loopback 1.x) uses `route`
+    var explorerPath = explorerApp.mountpath || explorerApp.route;
+    console.log('Browse your REST API at %s%s', baseUrl, explorerPath);
+  });
+};
```

Comment:
We use slightly different project layout now and both explorer & rest api are loaded via JSON files (see https://github.com/strongloop/loopback-workspace/pull/230). Could you please update the configuration of the test project to follow the current convention?